client/aliases.go

```diff
@@ -26,9 +26,12 @@ import (
 
 // aliasAction represents an action performed on aliases.
 type aliasAction struct {
-	Action  string   `json:"action"`
-	Snap    string   `json:"snap"`
-	Aliases []string `json:"aliases"`
+	Action string `json:"action"`
+	Snap   string `json:"snap"`
+	Target string `json:"target,omitempty"`
```

Comment:
Wouldn't that be more clear as `"App"/"app"`? Looking at the rest of the PR, it feels like that's what we talk about everywhere else.